examples/basics/plotting/grid_uneven_col.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
```

Comment:
but if you modify `stretch` to auto-depend (by default) on `colspan` and `rowspan`, then you won't need `stretch` in either example, only `*_span`